EvidenceApi.Tests/V1/Controllers/DocumentTypesControllerTests.cs

```diff
@@ -70,20 +199,40 @@ public void ReturnsStaffSelectedDocumentTypes()
         }
 
         [Test]
-        public void ReturnNotFoundIfEmptyStaffSelectedDocumentTypes()
+        public void ReturnsEmptyListWhenNoStaffSelectedDocumentTypes()
```

Comment:
To clarify, it is now possible to have the following scenarios:

search for incorrect team --> returns no documents --> return NotFound
search for correct team with query --> returns no documents --> return empty array
This is because in the first instance, the 404 is a client error. In the second instance, everything is correct, there is just no data.